src/components/layout/nav.js

```diff
@@ -116,6 +116,9 @@ export const Wrapper = styled(Container)`
     @media ${device.laptop} {
         font-size: var(--text-size-xxs);
     }
+    @media ${device.mobileM} {
+        width: 98%;
```

Comment:
Could this be replaced with a `calc(100%-offset_px_required)` or it is more suitable in this case for it to be in a specific `%`?

Reply:
updated. thanks :)